setup/baseConfig.yaml

```diff
@@ -27,10 +27,6 @@
 #  10. 'sdk-trace' has been removed with v2.3.0 release and setting log level to log_debug will auto enable these logs.
```

Comment:
Should we also rename baseConfig to simplifiedConfig or something else?

Reply:
I wanted to change it to basicConfig since its similar to baseConfig? I'm not sure because most customers might be accustomed to looking for baseConfig. What do you think about this?

Reply:
baseConfig name is fine. Let' remove the comments above from line 1-28, since these are already present in the advanced config

Reply:
Cool